test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -34,6 +34,27 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, 1, Integer.valueOf(1));
         assertNotUnique(null, null);
         assertNotUnique(null, "a", "b", null);
+
+    }
+
+    @Test
+    public void elementsAreNull() throws Exception {
```

Comment:
It is better to follow the convention that the names of the method being tested and the test method are the same, at least for the sake of better readability and tracking.